functionary/core/models/package.py

```diff
@@ -106,3 +123,8 @@ def full_image_name(self) -> str:
     def active_functions(self) -> QuerySet:
         """Returns a QuerySet of all active functions in the package"""
         return self.functions.filter(active=True)
+
+    @property
+    def enabled(self) -> bool:
```

Comment:
Commented elsewhere, but it feels like for the sake of consistency this should be the same as whatever we named the function version of this, which is currently `is_active`.